app/views/layouts/application.html.erb

```diff
@@ -10,7 +10,7 @@
     <%= javascript_importmap_tags %>
 
     <%= render "shared/cdn_assets" %>
-
+   
     <style>
```

Comment:
this code probably belongs better in the application css file since this is technically a html file